svf/lib/MSSA/MemRegion.cpp

```diff
@@ -461,12 +461,11 @@ bool MRGenerator::addModSideEffectOfCallSite(const CallICFGNode* cs, const NodeB
  */
 void MRGenerator::getCallGraphSCCRevTopoOrder(WorkList& worklist)
 {
-
-    NodeStack& topoOrder = callGraphSCC->topoNodeStack();
-    while(!topoOrder.empty())
+    NodeStack revTopoNodeStack = callGraphSCC->revTopoNodeStack();
```

Comment:
@canliture I think we could just revise this method by using one line: `worlist = callGraphSCC->revTopoNodeStack();`

could you validate the correctness again?